data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES (1, 'Agumon','feb/3/2020',0,true,10.23);
+INSERT INTO animals VALUES (2, 'Gabumon','nov/15/2018',2,true,8);
+INSERT INTO animals VALUES (3, 'Pikachu','jan/7/2021',1,false,15.04);
+INSERT INTO animals VALUES (4, 'Devimon','may/12/2017',5,true,11);
```

Comment:
- [ ] We are missing the creation / and / or adding required fields to table animal, whereby the following fields are required:
```sql
    CREATE TABLE animals (
    id int,
    name varchar(255),
    date_of_birth date,
    escape_attempts integer,
    neutered boolean,
    weight_kg decimal,
    );
```
Would you add required fields to the table animal?

Reply:
- [ ] The insert query needs to specify which fields to which is inserting data, for example:
```sql
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23');
```
Would you adjust your insert queries according to the given example above?

Reply:
- [ ] [optional] I would suggest you to ensure that the dates you are providing are of valid format, so that when we run insert queries against your db, we won't get a crash errors

Reply:
for the record, I have to say If you don't specify the fields it will automatically add to the fields according to your table structure. I don't know why you are insisting to make my work harder by explicitly writing each field. though I don't need to.

Reply:
I have added the part to create the animal table in the schema.sql according to the requirements it should be added them not in data.sql. I suggest you to see the project requirements before reviewing any project.

Reply:
Hello @shadabwahidullah about this: 
1. 
> I have added the part to create the animal table in the schema.sql according to the requirements it should be added > > them not in data.sql. I suggest you to see the project requirements before reviewing any project.

I know about the requirements and by the time I reviewed your work you had only this line of code in `schema.sql` file: 
```sql
CREATE TABLE animals (name varchar(100))
```
👍  Thank you for addressing this issue for now! ✔️ 

2.
>for the record, I have to say If you don't specify the fields it will automatically add to the fields according to your table > structure. I don't know why you are insisting to make my work harder by explicitly writing each field. though I don't > > need to.

I understand what you meant here and I know that it works well, I insisted on this for you to master the best practices, and when you master best practices, will limit you committing errors in your codes, it's very good and best practice to explicitly specify the fields you are to which you are inserting data.

👍 Thank you for addressing this issue ✔️